src/Symfony/Component/Translation/Resources/functions/translatable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Translation\Translatable;
+
+if (!function_exists('_t')) {
+    /**
+     * @author Nate Wiebe <nate@northern.co>
+     */
+    function _t(string $messageId, array $parameters = [], string $domain = 'messages'): Translatable
```

Comment:
Is is possible to give this function a full name? `_t()` is in used for the application I work in and has 1000+ results when searching for it.

Reply:
we have `s()` also, the full name is `new Translatable` IMHO